File: Tavisca.Applause/Models/Employee.cs
namespace Tavisca.Applause
{
    public class Employee
    {
       
        public string Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Tribe { get; set; }
        public string Squad { get; set; }
        public string ProfilePictureUrl { get; set; }
        public bool IEnabled { get; set; }
        public string Email { get; set; }
        public EmployeeRewardsStats RewardStats { get; set; }
    }
}

File: Tavisca.Applause.Web/ClientApp/src/app/shared/models/dashboard/employee.ts
export class RewardStats {
  employeeId: string;
  points: number;
  bronzeBadge: number;
  silverBadge: number;
  goldBadge: number;
  annual: number;
  kudos: number;
  shoutOut: number;
}

export class Employee {
  id: string;
  firstName: string;
  lastName: string;
  tribe: string;
  squad: string;
  profilePictureUrl: string;
  iEnabled: boolean;
  email?: any;
  rewardStats: RewardStats;
}
